.devcontainer/Dockerfile

```diff
@@ -0,0 +1,22 @@
+#-------------------------------------------------------------------------------------------------------------
```

Comment:
Just wondering about the whole `.devcontainer` directory, what's the point of it?